src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -102,32 +104,39 @@ def _validate_start_time(start_time, end_time):
     return result_time
 
 
-# pylint: disable=too-many-arguments
-def list_activity_logs(client, correlation_id=None, resource_group=None, resource_id=None,
-                       resource_provider=None, start_time=None, end_time=None,
+# pylint: disable=too-many-arguments, line-too-long
```

Comment:
It's coming from line 132 
`if len([x for x in [correlation_id, resource_group, resource_id, resource_provider] if x]) > 1:` 
if i split it i think it'd be not readable.

Or i can create local list for `[correlation_id, resource_group, resource_id, resource_provider]` to decrease characters on line 132